app/views/comments/_index.html.erb

```diff
@@ -0,0 +1,43 @@
+<section class="add-comment">
```

Comment:
Maybe this block better to place in `section.comments`? And remove duplication of classes in a [4 line](https://github.com/EvgeniiK/Mugenblog/pull/36/files#diff-53e600e3c9b5ce6a6226b7245ec587f3R4) of a `comments.scss` file.